paddle/framework/op_desc.proto

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+syntax="proto2";
+package paddle.framework;
+
+import "attr_type.proto";
+
+// AttrDesc is used to set Attrubute for Operator. It contain's
+// name, type and the value of this Attrubute.
+message AttrDesc {
+    required string name = 1;
+    required AttrType type = 2;
+    optional int32 i = 3;
+    optional float f = 4;
+    optional string s = 5;
+    repeated int32 ints = 6;
+    repeated float floats = 7;
+    repeated string strings = 8;
+};
+
+// OpDesc(Operator Description) is used to describe and construct
+// a Operator. 3rd-party language can construct this proto message
+// and call AddOp(const OpDesc* op_desc) to create a Operator in Paddle
+// core.
+message OpDesc {
+    // input names of this Operator.
+    repeated string inputs = 1;
+
+    // output names of this Operator.
+    repeated string outputs = 2;
+
+    // type of the Operator, such as "ADD" "SUB".
+    required string type = 3;
+
+    // some Attribute of this Operator.
```

Comment:
`attributes of current operator. e.g., `scale=3.0` in `cosine op`.`

Add an example in comments.